WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -108,4 +108,19 @@ + (NSDictionary *)accessibilityInfoForElement:(XCElementSnapshot *)snapshot
   return info;
 }
 
+-(void) fb_resolve
+{
+  if (self.fb_isActivateSupported) {
+    
+    if(self.state != XCUIApplicationStateRunningForeground)
+    {
+      [self activate];
+    }
```

Comment:
is it expected that resolve should not be called if `self.state == XCUIApplicationStateRunningForeground` ?

Reply:
Yes